### ISSUE TITLE
Sketch Mobile templates out of date

### ISSUE THREAD

**Issue by jeana-adhoc** [department-of-veterans-affairs/vets-design-system-documentation#1643]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

In the sketch library - there is a `Templates > Mobile` that has the header and footer in one symbol, but then there's also a`Navigation > header > mobile > authenticated` symbol. The headers do not match in these two instances (But the footers do). 

![image](https://user-images.githubusercontent.com/117098918/228261350-6c124228-d685-4759-8062-c89c8540587f.png)

## What I expected to happen

The headers and footers to match in the template and separate navigation symbols

## Urgency
How urgent is this request? Please select the appropriate option below and/or provide details
- [ ] This bug is blocking work currently in progress
- [X] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [ ] This bug is not blocking any work
- [ ] Other

**Comment by humancompanion-usds** [comment]
@jeana-adhoc - I've fixed all of the templates on all of the pages excluding "Form templates", which has the most artboards and is still in progress, to use the correct headers from our Sketch library at each viewport size. I'm making progress but that set of Form template is a doozy. My goal is to not have Templates have any symbols for headers at the very least.

**Comment by humancompanion-usds** [comment]
Oh, and @jeana-adhoc, I just realized you meant in the Sketch component library there was a Templates symbol. I didn't even realize that was there. I wonder if we should delete it. We've got a whole library of Templates: https://sketch.com/s/c8df169f-5b02-4999-befb-34c7b3b62ba9 Is the symbol useful?

**Comment by humancompanion-usds** [comment]
Okay, I reorganized the form templates and realized our pattern and form-related Sketch templates are quite out-of-date. I'm going to close this issue because I've updated all of the headers and footers in the Templates library. I'm going to talk to @jeana-adhoc about becoming the maintainer of a [Shared Library](https://design.va.gov/about/designers/design-libraries#shared-libraries) related to forms.
